src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -3994,6 +4050,235 @@ module Mutations = struct
+                            in
+                            let body =
+                              Signed_command_payload.Body.Payment
+                                { source_pk
+                                ; receiver_pk
+                                ; amount = payment_details.amount
+                                }
+                            in
+                            let valid_until = None in
+                            let nonce = nonces.(ndx) in
+                            let payload =
+                              Signed_command_payload.create ~fee
+                                ~fee_payer_pk:source_pk ~nonce ~valid_until
+                                ~memo ~body
+                            in
+                            let signature =
+                              Ok (Signed_command.sign_payload sender payload)
+                            in
+                            [%log info]
+                              "Payment scheduler with handle %s is sending a \
+                               payment from sender %s"
+                              (Uuid.to_string uuid)
+                              ( Signature_lib.Public_key.Compressed.to_yojson
+                                  source_pk
+                              |> Yojson.Safe.to_string )
+                              ~metadata:
+                                [ ( "receiver"
+                                  , Signature_lib.Public_key.Compressed
+                                    .to_yojson receiver_pk )
+                                ; ("nonce", Account.Nonce.to_yojson nonce)
+                                ; ("fee", Currency.Fee.to_yojson fee)
+                                ; ( "amount"
+                                  , Currency.Amount.to_yojson
+                                      payment_details.amount )
+                                ; ("memo", `String payment_details.memo)
+                                ] ;
+
+                            let%bind () =
+                              let fee = Currency.Fee.to_uint64 fee in
+                              let memo = Some payment_details.memo in
+                              match%bind
+                                send_signed_user_command ~mina
+                                  ~nonce_opt:(Some nonce) ~signer:source_pk
+                                  ~memo ~fee ~fee_payer_pk:source_pk
+                                  ~valid_until ~body ~signature
+                              with
+                              | Ok _cmd_with_status ->
+                                  Deferred.unit
+                              | Error _ ->
+                                  Deferred.unit
+                            in
+                            (* next nonce for this sender *)
+                            nonces.(ndx) <- Account.Nonce.succ nonce ;
+                            let%bind () = Async_unix.after wait_span in
```

Comment:
Should this function emit multiple txns when the time elapsed is more than `wait_span` by some multiple? Otherwise, the txns per second will be less than the target, determined by the load against the async scheduler.

Reply:
I'll see if I can compensate for that.